warehouse/forklift/legacy.py

```diff
@@ -251,8 +251,11 @@ def _validate_project_url(value):
 
 
 def _validate_project_url_list(form, field):
-    for datum in field.data:
-        _validate_project_url(datum)
+    if isinstance(field.data, str):
+        _validate_project_url(field.data)  # Or raise ValidationError?
```

Comment:
@dstufft i'm not sure if we want to just accept a string or not here.

Reply:
`Project-URL` is a multiple-use field, so this should always be a list. I think it should just fail validation if it isn't.

Reply:
oh, you're darn tootin.

https://github.com/pypa/warehouse/blob/6fe519b7506702bdf961631bdb2373670842bf21/warehouse/forklift/legacy.py#L464-L469